helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -575,6 +575,17 @@ logsCollection:
     # Set to 0 to remove any size limit.
     maxRecombineLogSize: 1048576
 
+    # Regex to extract logs, time for CRI-O or containerd logs
+    # More info: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/operators/regex_parser.md
+    regexp: ""
```

Comment:
A Kubernetes environment can use different container runtimes on different nodes at the same time. The suggested override configurations should be granular enough to configure each parser (crio, containerd, docker) individually.

Reply:
Okayy, will take care of this one and update the PR